packages/modelence/src/system/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Module } from '../app/module';
+
+export default new Module('_system', {
```

Comment:
@omegascorp I'm not sure how I feel about having a `_system` module. I think one Module definitely shouldn't be a prefix of another - it can lead to weird issues in the future. Maybe we can just make configs global and not force them into the Module structure?

Reply:
We don't have a proper way of making configs global, but this works for now, it's the easiest way to define configs without schema (it it's not something that will concern the users, so we can change this in the future if we define global configs)